src/main/java/com/leverx/learningmanagementsystem/config/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,90 @@
+package com.leverx.learningmanagementsystem.config.security;
+
+import com.sap.cloud.security.xsuaa.XsuaaServiceConfiguration;
+import com.sap.cloud.security.xsuaa.token.TokenAuthenticationConverter;
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+@AllArgsConstructor
+public class SecurityConfiguration {
+
+    XsuaaServiceConfiguration xsuaaServiceConfiguration;
+
+    @Bean
+    @Profile("cloud")
+    @Order(1)
+    public SecurityFilterChain actuatorSecurityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf(AbstractHttpConfigurer::disable)
+                .securityMatcher("/actuator/**")
+                .authorizeHttpRequests(request -> request
+                        .anyRequest().hasRole("MANAGER"))
+                .httpBasic(Customizer.withDefaults());
+        return http.build();
+    }
+
+    @Bean
+    @Profile("cloud")
+    public SecurityFilterChain apiSecurityFilterChain(HttpSecurity http) throws Exception {
+        http
```

Comment:
the same is here, it's requried to define path even it's equal to "/**" (which is used by default is nothing is mentioned), just for easier understanding for which requests this `SecurityFilterChain` is responsible for:
```
http.securityMatcher("/**")
    .csrf(...)
    ...
```